src/openrct2/paint/track/coaster/LatticeTriangleTrackAlt.cpp

```diff
@@ -194,6 +194,21 @@ static void LatticeTriangleTrackAltPoweredLift(
     PaintUtilSetGeneralSupportHeight(session, height + 56);
 }
 
+static void LatticeTriangleTrackAltDiagBooster(
+    PaintSession& session, const Ride& ride, uint8_t trackSequence, uint8_t direction, int32_t height,
+    const TrackElement& trackElement, SupportType supportType)
+{
+    constexpr ImageIndex images[2][kNumOrthogonalDirections] = {
+        { SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_ALT_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_ALT_2,
+          SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_ALT_3, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_ALT_4 },
+        { SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2,
+          SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2 },
```

Comment:
Idem.

Reply:
Same deal here. Remove the duplicate top-level array so it's only a single dimensional array, and send the entire images array instead of the subset.